pkg/generator/generator_test.go

```diff
@@ -105,9 +105,19 @@ func TestGenerator_generateRawConfig(t *testing.T) {
 		},
 
 		{
-			name:                 "case 9 - throw error when a key is missing",
+			name:                 "case 9 - throw error when secret values are undefined",
 			caseFile:             "testdata/case9.yaml",
-			expectedErrorMessage: `<.this.key.is.missing>: map has no entry for key "this"`,
+			expectedErrorMessage: `Empty value: value is undefined: "undefinedValue"`,
```

Comment:
I expected to find `{{ .undefinedValue }}` in the template. My suggestion would be to print something like : `Empty value: value for key "undefinedValue" is undefined`